backend/handlers/handlers.go

```diff
@@ -750,12 +750,24 @@ func (handlers *Handlers) getAccountsBalance(*http.Request) (interface{}, error)
 	return totalAmount, nil
 }
 
+type coinFormattedAmount struct {
+	CoinCode        coin.Code                       `json:"coinCode"`
+	CoinName        string                          `json:"coinName"`
+	FormattedAmount accountHandlers.FormattedAmount `json:"formattedAmount"`
+}
+
+type coinBalance struct {
+	CoinCode     coin.Code
+	TotalBalance *big.Int
+}
+
 // getCoinsTotalBalance returns the total balances grouped by coins.
 func (handlers *Handlers) getCoinsTotalBalance(_ *http.Request) (interface{}, error) {
-	totalPerCoin := make(map[coin.Code]*big.Int)
 	conversionsPerCoin := make(map[coin.Code]map[string]string)
 
-	totalAmount := make(map[coin.Code]accountHandlers.FormattedAmount)
+	var coinFormattedAmounts []coinFormattedAmount
+	var totalsPerCoin []coinBalance
```

Comment:
Since you will store the balances inside `totalCoinsBalances` you don't need to define the `coinBalance` type. This can just be a coin.Code slice. Also, I'd rename it to highlight the fact that it is used to keep track of the sorting: something like `sortedCoins` or `sortedCoinCodes` maybe?